### PR TITLE
Prepare release 2.55.0-rc.1

### PR DESCRIPTION
**Pull request by bboreham**
Two bugfixes, plus experimental `info` function.

### CODE REVIEW

**Comment by bboreham** [comment]
Wait we need #14947 too.

**Comment by bboreham** [comment]
I would say #15120 should also be included, but could be another point release. 

I won’t be able to do the release for a few days; if you want to do it please go ahead.

**Comment by bboreham** [comment]
#15120 now backported.

Holding out to include #14495 so the MVP can get feedback sooner.

**Review by aknuds1 — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,7 +2,10 @@
 
 ## unreleased
```
Comment (machine424) [inline CHANGELOG.md:4]: I don't see an entry for delayed-compaction https://github.com/prometheus/prometheus/pull/12532#issuecomment-2407032854

**Pull request merged**